.github/workflows/ci.yaml

```diff
@@ -63,6 +65,7 @@ jobs:
       path-to-charm-directory: ${{ matrix.charm.path }}
       charmcraft-snap-channel: ${{ matrix.charm.charmcraft }}
       cache: ${{ matrix.charm.cache }}
+      artifact-prefix: temp-packed-charm-${{ matrix.charm.name }}
```

Comment:
Without the name variable, I got conflicts between the test and data integrator charms, both tried to use the same artefact name.